docs/span_attribute_schema/v0.md

```diff
@@ -155,6 +155,28 @@ http.status_code | Yes
 http.url | Yes
 span.kind | `client`
 
+## AwsSns
+### Span properties
+Name | Required |
+---------|----------------|
+Name | `sns.request`
+Type | `http`
+### Tags
+Name | Required |
+---------|----------------|
+aws.agent | `dotnet-aws-sdk`
+aws.operation | Yes
+aws.region | No
+aws.requestId | Yes
+aws.service | `SNS`
+aws.topic.arn | No
+aws.topic.name | No
+component | `aws-sdk`
+http.method | Yes
+http.status_code | Yes
+http.url | Yes
+span.kind | `client`
+
```

Comment:
For some reason this hadn't been generated before, and I can see no reason for it to be excluded

Reply:
It may conflict with one of @zacharycmontoya's PR as he added it as well. So whoever merge first wins :)